pkg/reconciler/pipelinerun/cancel_test.go

```diff
@@ -38,196 +38,15 @@ import (
 
 func TestCancelPipelineRun(t *testing.T) {
 	testCases := []struct {
-		name           string
-		embeddedStatus string
-		pipelineRun    *v1beta1.PipelineRun
-		taskRuns       []*v1beta1.TaskRun
-		runs           []*v1alpha1.Run
-		customRuns     []*v1beta1.CustomRun
-		wantErr        bool
+		name string
+
+		pipelineRun *v1beta1.PipelineRun
+		taskRuns    []*v1beta1.TaskRun
+		runs        []*v1alpha1.Run
+		customRuns  []*v1beta1.CustomRun
+		wantErr     bool
 	}{{
```

Comment:
I don't think all of these test cases can be deleted; we need to make sure all these cases are still covered. For example, I don't think there's currently a test case for "no resolved taskrun", since "no-resolved-taskrun-with-full" was deleted. Can you take a closer look at these to figure out which are redundant without an embedded-status flag, and which need to be updated to use childreferences?

Reply:
restored, sorry that I thought the ones were relied on taskruns status.